miniupnpd/upnppermissions.c

```diff
@@ -172,15 +305,61 @@ read_permission_line(struct upnpperm * perm,
 	{
 		return -1;
 	}
+
+	/* fifth token: (optional) regex */
+	p = get_next_token(p, &perm->re, 1);
```

Comment:
why is get_next_token() needed ?
Isn't taking the end of the line OK ?

Reply:
so that forward compatibility with more option tokens is possible.